frontend/packages/metal3-plugin/src/components/machine-cell.tsx

```diff
@@ -34,7 +36,7 @@ const MachineCell: React.FC<MachineCellProps> = ({ host }) => {
       />
     );
   }
-  if (canHostAddMachine(host)) {
+  if (status.status === HOST_STATUS_READY) {
```

Comment:
Same as above, why not keeping the helper function? We could change it so it takes hostStatus as parameter rather than a host.